src/models/train_model.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+import torch
+import torch.nn as nn
+from torch.optim import Adam
+from torch.utils.tensorboard import SummaryWriter
+
+from network import Model
+from training import *
+
+import click
+import logging
+from pathlib import Path
+from dotenv import find_dotenv, load_dotenv
+
+
+@click.command()
+@click.argument('input_features', type=click.File('rb'))
+@click.argument('input_target', type=click.File('rb'))
+def main(input_features, input_target):
+    """ Runs model training scripts to create and train models on prepared
+        datasets (from ../processed) and save them in ../models.
+    """
+    logger = logging.getLogger(__name__)
+    
+    # Get reproducible results
+    torch.manual_seed(42)
+
+    writer = SummaryWriter()
+
+    logger.info('loading data')
+    X = np.genfromtxt(input_features, delimiter=',')
+    y = np.genfromtxt(input_target, delimiter=',')
+    X_train, X_test, y_train, y_test = data_split(X, y)
+
+    logger.info('running')
+    # Set up network and training parameters
+    model = Model(X_train.shape[1], len(np.unique(y_train)), (100, 240, 10), 'relu', 'softmax')
```

Comment:
We will do a lot of hyper-parameter tuning, make sure from the very beginning that the executable code in this module is  easy for parametrization. Instead of hard-coding model parameters in the invocation of the constructor, please create a separate `Configuration` class (of course, name it as you see fit) and make sure that it is easy to provide multiple configurations to model training (think of running a grid search on parameter space)